runtime/server_http_response.go

```diff
@@ -137,6 +138,17 @@ func serverHTTPLogFields(req *ServerHTTPRequest, res *ServerHTTPResponse) []zapc
 		zap.Int("statusCode", res.StatusCode),
 	}
 
+	if span := req.GetSpan(); span != nil {
+		jc, ok := span.Context().(jaeger.SpanContext)
+		if ok {
+			fields = append(fields,
+				zap.String("trace.span", jc.SpanID().String()),
```

Comment:
Another option was this (below) since span ids are `uint64s`, however I kept it as string for convenience.
```
+ zap.Uint64("trace.span", uint64(jc.SpanID()))
- zap.String("trace.span", jc.SpanID().String()),
```